### PR TITLE
Add support for presence capability

### PR DESCRIPTION
**Pull request by ScottChapman**
Added action and supporting script for presence devices

### CODE REVIEW

**Comment by ScottChapman** [comment]
fixes #86

File: smartapps/stj/mqtt-bridge.src/mqtt-bridge.groovy [hunk 1]
```diff
@@ -659,6 +660,16 @@ def actionLevel(device, attribute, value) {
     device.setLevel(value as int)
 }
 
+def actionPresence(device, attribute, value) {
+	log.info("Setting presence");
+	if (value == "present") {
```
Comment (stjohnjohnson) [inline smartapps/stj/mqtt-bridge.src/mqtt-bridge.groovy]: Please fix the indentation here.

**Comment by stjohnjohnson** [comment]
Thanks for this PR.  I don't have a presence device, so I can't fully test this.  But it looks reasonable.  Can you just fix the indentation in the function?

**Comment by ScottChapman** [comment]
It is actually easy to test with the "Simulated Presence Sensor" that smartthings provides (that's actually what I am using for this)

**Comment by stjohnjohnson** [comment]
Thanks!!

**Pull request merged**